app/src/main/java/com/rentminder/Icon&Text.kt

```diff
@@ -43,7 +38,7 @@ fun ElectricIconText() {
         ),
             contentDescription = "Light Bulb Icon",
             modifier = Modifier
-                .size(30.dp)
+                .size(30.dp),
```

Comment:
Added comma as per the kotlin best practices
https://kotlinlang.org/docs/coding-conventions.html